pkg/v1/cluster/schemas.go

```diff
@@ -150,9 +150,13 @@ func (result *View) UnmarshalJSON(b []byte) error {
 }
 
 // KubernetesOptions represents additional k8s options such as pod security policy,
-// feature gates and etc.
+// feature gates (Alpha stage only) and admission controllers.
 type KubernetesOptions struct {
 	// EnablePodSecurityPolicy indicates if PodSecurityPolicy admission controller
 	// must be turned on/off.
 	EnablePodSecurityPolicy bool `json:"enable_pod_security_policy"`
+	// FeatureGates represents feature gates that should be enabled.
```

Comment:
Please add a blank line before the comment similar to the code style used in the package.
This remark applies for this and the following comment.